### PR TITLE
fix stuff on settings page

### PR DESCRIPTION
**Pull request by tron975**

### CODE REVIEW

**Comment by Polydynamical** [comment]
Can you post a screenshot of how this looks like?

File: css/style.css [hunk 1]
```diff
@@ -177,13 +177,13 @@ background: url(https://cdn.artofproblemsolving.com/attachments/5/8/0e3aad833ef3
...
 
-.modalButton:focus {
+.modalButton:focus-within {
   background-color: white;
   opacity: 0.9;
 }
 
 .modalButton:hover {
-  background-color: white;
+  background-color: rgb(132, 243, 88);
```
Comment (Polydynamical) [inline css/style.css:186]: why green?

File: css/style.css [hunk 2]
```diff
@@ -236,10 +237,11 @@ background: url(https://cdn.artofproblemsolving.com/attachments/5/8/0e3aad833ef3
   padding: 8px 16px;
   transition: 0.2s;
   width: 150px;
+  border-bottom-left-radius: 5px;
+  border-bottom-right-radius: 5px;
```
Comment (Polydynamical) [inline css/style.css:241]: very nice

File: play.html [hunk 3]
```diff
@@ -62,7 +62,7 @@ <h2>Credits</h2>
                 <header class="container tab">
                     <span onclick="closeModal()" style="position: absolute; right: 0; top: 0;" class="modalButton" id = "close"></span>
                 </header>
-                <div style="border-bottom: 1px solid #ccc">
+                <div style="border-bottom: 1px solid #ccc" class = "tab-container">
```
Comment (Polydynamical) [inline play.html:65]: remove spaces before and after "=" sign

File: css/style.css [hunk 4]
```diff
@@ -392,4 +394,11 @@ p {
   fill: red;
   stroke: rgb(255, 252, 84);
   transform: scale(0.96, 0.96);
+}
+
+.tab-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 2rem;
```
Comment (Polydynamical) [inline css/style.css:403]: this looks good but send screenshot so I can see how it looks like

**Review by Polydynamical — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 1 above)
Comment (tron975) [reply css/style.css:186]: Its just a darker green than the original light green

File: css/style.css (hunk 4 above)
Comment (tron975) [reply css/style.css:403]: image

File: css/style.css (hunk 1 above)
Comment (Polydynamical) [reply css/style.css:186]: 👍

**Review by Polydynamical — APPROVED** [review]

**Pull request merged**